### ISSUE TITLE
Conflicts with default category assignee

### ISSUE THREAD

**Issue by majkinetor** [giddie/redmine_default_assign#30]
I thought interactive mode will solve this, but it still happens -> default assignee will overwrite default category assignee.

**Comment by giddie** [comment]
Yep, good point. I guess the category assignee is applied after the plugin's hook is called.

**Comment by majkinetor** [comment]
Category assignee is never visible until you commit the issue. This plugin sets it up ASAP. In my company this makes it unusable since every project has bunch of categories with default assignees and we wanted to use default one only if category is not specified or it doesn't have a default assignee itself.

**Comment by giddie** [comment]
Can you explain in more detail what you mean when you say "this plugin sets it up ASAP"?

**Comment by majkinetor** [comment]
I mean the assignee is visible as soon as you click New Issue. Contrary to that assignee is empty with category defaults and becomes visible once you save it

**Comment by giddie** [comment]
Have you tried changing the "interactive" option in the settings for the plugin?

**Comment by majkinetor** [comment]
Yup, nothing changed.

**Comment by giddie** [comment]
OK, so just to clarify: you're saying that the assignee is filled in already when the "New Issue" form is shown?

**Comment by majkinetor** [comment]
Yes.

**Comment by majkinetor** [comment]
My bad. I rechecked again and it works with "non interactive" settings. With interactive it doesn't.

**Comment by giddie** [comment]
OK, good. I'm not going crazy then :p

**Comment by giddie** [comment]
I'll leave this open to look into how easy it might be to account for category assignees in interactive mode.

**Comment by majkinetor** [comment]
Thx for your help
